src/main/java/gregtech/common/tileentities/machines/multi/drone/MTEHatchDroneDownLink.java

```diff
@@ -158,19 +178,19 @@ public boolean allowPutStack(IGregTechTileEntity aBaseMetaTileEntity, int aIndex
 
     @Override
     public void onRemoval() {
-        if (hasConnection()) connection.machine = null;
+        center.getConnectionList()
```

Comment:
This crashes when breaking a hatch that's not connected to a Drone Centre.
```suggestion
        if (center == null) return;
        center.getConnectionList()
```